test/records-test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * records-test.js - Records tests for hsd
+ * Copyright (c) 2019, Mark Tyneway (MIT License).
+ * https://github.com/handshake-org/hsd
+ */
+
+'use strict';
+
```

Comment:
@boymanjor If I port this file to another PR without the cosmos additions, do you think we could merge it in? Having the tests in would start the `records-test` file and allow it to be added to without needing this PR to be merged. I could rebase the cosmos additions back on top and then go from there.